README.md

```diff
@@ -49,6 +49,14 @@ Define the type of `{{ . }}` in the context of your template by adding a line li
 
 `make cypress`
 
+To see the UI output you can still run 
+
+`yarn && yarn cypress`
+
+If you change anything in the code run the below to update the image you are testing against
+
+`make up`
```

Comment:
Working in this way shouldn't be the default. `make dev-up` runs the Go app in with a debugger and hot reloading, so you shouldn't be running anything to update the image.